code/processes/gateway.q

```diff
@@ -41,13 +41,13 @@
 // each error will be prefixed with the errorprefix (currently "error: ")
 // the client should check if the result is a string, and if it is if it is prefixed with "error: "
 // errors will be returned when 
```

Comment:
I think this comment section should be update to reflect the 3.6 changes - i.e. the new functionality we now have